go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	cuelang.org/go v0.2.2
+	github.com/Azure/go-autorest v12.2.0+incompatible // Don't remove. https://github.com/kubernetes/client-go/issues/628
```

Comment:
Can it be fixed by https://github.com/kubernetes/client-go/issues/628#issuecomment-569683255 ?

Reply:
No. These two lines doesn't show in go.mod.
```
github.com/Azure/go-autorest/autorest v0.9.2 // indirect
github.com/Azure/go-autorest/autorest/to v0.3.0 // indirect
```
They are not direct dependency.

Reply:
ok